cli/aws_orbit/remote_files/cdk/lambda_sources/eks_service_handler/index.py

```diff
@@ -0,0 +1,72 @@
+#  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License").
+#    You may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+
+import json
+import logging
+import os
+from typing import Any, Dict, List, Optional
+
+import boto3
+
+logger = logging.getLogger()
+logger.setLevel(logging.INFO)
+
+REGION: str = os.environ["REGION"]
+
+
+def get_nodegroups(cluster_name: str) -> List[Dict[str, Dict[str, str]]]:
+    return_response: List[Dict[str, Dict[str, str]]] = []
+    eks_client = boto3.client("eks")
+    ec2_client = boto3.client("ec2")
+    try:
+        nodegroups_response = eks_client.list_nodegroups(clusterName=cluster_name)
+        for nodegroup_name in nodegroups_response["nodegroups"]:
```

Comment:
should we do so much in the lambda ?  or just return almost what the list* gives you and do all that inside the client code?

Reply:
agreed. just returning the sdk response will reduce the amount of rework required when the vpc endpoint becomes available

Reply:
Agreed. If we move list_nodegroups to client code, need to trigger lambda twice. 1) list_nodegroups 2) describe_nodegroup. 

ec2..describe_launch_template_versions can be moved to client code to use VPC endpoint.

Reply:
ok, I get it , I guess its not worth splitting into multiple calls, but whatever else can be reduced would be good.

Reply:
Moved ec2 calls to SDK. Lambda holds eks calls.